### ISSUE TITLE
Confusing error message

### ISSUE THREAD

**Issue by kach** [accordproject/ergo#302]
```
ergotop$ define function f (x : Integer) : Integer { return 5 }
ergotop$ f(4);
Compilation error at 1:0-1:4 'Cannot use 'contract' variable outside of a contract'
```

What's going on? This is definitely a thing that users will try to do (forget to use `return`). There is no mention of "contract" here, though…

**Comment by jeromesimeon** [comment]
This is confusing, not sure it's just a confusing error message. Could be a bug. Does `return f(4);` work?

**Comment by kach** [comment]
Yeah, `return f(4);` works as expected.

```
ergotop$ define function f (x : Integer) : Integer { return 5 }
ergotop$ return f(4);
Response. {"nat": 5.0}
```

**Comment by kach** [comment]
Maybe it's parsing it as a "clause call"?

**Comment by jeromesimeon** [comment]
Yes! See also: #277

**Comment by jeromesimeon** [comment]
Resolved, remaining of this discussion in #277
